sources/install/package_osint.sh

```diff
@@ -284,10 +284,23 @@ function install_maltego(){
     colorecho "Installing Maltego"
     wget https://maltego-downloads.s3.us-east-2.amazonaws.com/linux/Maltego.v4.3.0.deb -O /tmp/maltegov4.3_package.deb
     dpkg -i /tmp/maltegov4.3_package.deb
-    # TODO add-test-command
+    add-test-command "file /usr/share/maltego/bin/maltego"
```

Comment:
Isn't there another way to test if maltego can run ?

Reply:
The tool is entirely based on graphical display. I haven't found a workable solution. Can we make a ticket to manage the tests on the graphic tools for later ?

Reply:
No idea how we'll test those GUI-based tools